### ISSUE TITLE
Blocks with a wide range of drops are poorly supported

### ISSUE THREAD

**Issue by TheDeviantCrafter** [way2muchnoise/JustEnoughResources#177]
Thaumcraft's urns drop a wide variety of potions, along with a few other items like diamonds and ender pearls. To get a good representitive sample of what this block drops, you'd need to break a few hundred of them.

I copied profiled a *huge* amount of terrain, but only a handful of potions are listed. This is because urns have a wide range of drops, but generate rarely.

I'd like to be able to add a list of blocks to a config. JER would list the frequency of the specific block as it does now, but it would check for block drops in a more thorough manner. It should either calculate the absolute percentages from the loot table, or just randomly grab a few hundred items.

Urn screenshot:

![image](https://user-images.githubusercontent.com/18755309/54171778-a13de400-4449-11e9-9ec7-8923b75e663a.png)

**Comment by way2muchnoise** [comment]
Not 100% sure how I would implement this, I'm thinking a command to scan blocks (defined by blockstate) from a file. Something like `/jer_loot [count]` which would break and loot the blocks from a `lootables.txt` file. Would this work for you?

**Comment by TheDeviantCrafter** [comment]
Can you explain that in a little more detail? Would I put a list of blocks and blockstates in the `loot_tables.txt` file?

**Comment by way2muchnoise** [comment]
Yes, you would have to write down the blockstates of the items you would want to have "bruteforced". So adding `minecraft:sand` would have it bruteforce sand blocks. (FYI not 100% sure if this is even a possible option without actually putting it in the world.)
